pkgs/development/libraries/wlroots/default.nix

```diff
@@ -137,5 +138,19 @@ rec {
     ];
   };
 
-  wlroots = wlroots_0_17;
+  wlroots_0_18 = generic {
+    version = "0.18.0";
+    hash = "sha256-LiRnvu7qCbfSg+ONWVCtWwdzxxFZHfbgmy7zApCIW40=";
+    extraNativeBuildInputs = [
+      hwdata
+    ];
+    extraBuildInputs = [
+      ffmpeg
+      libliftoff
+      libdisplay-info
+      lcms2
+    ];
+  };
+
+  wlroots = wlroots_0_18;
```

Comment:
Are you sure you want to change the default immediately?

Reply:
Yes, other packages should depend on `wlroots_0_*`

https://github.com/NixOS/nixpkgs/pull/245704#pullrequestreview-1553286765

Reply:
That makes me wonder why have an unversioned attribute at all then.

Reply:
We typically used to follow Sway's wlroots version, and I think it still makes sense to expose wlroots-latest as an attribute, but we shouldn't use it internally. Maybe we can change it to an alias? Those are disalowed for use in nixpkgs afaik

Reply:
Will change it to an alias in https://github.com/NixOS/nixpkgs/pull/252816